src/js/views/globals/search/search-by-dob_views.js

```diff
@@ -0,0 +1,196 @@
+import { noop, find, get, map, debounce, each, extend, pick } from 'underscore';
+import dayjs from 'dayjs';
+import hbs from 'handlebars-inline-precompile';
+import { View, CollectionView } from 'marionette';
+
+import hasAllText from 'js/utils/formatting/has-all-text';
+
+import PicklistComponent from 'js/components/picklist';
+
+import InputWatcherBehavior from 'js/behaviors/input-watcher';
+import PicklistBehavior from 'js/behaviors/picklist-transport';
+
+const CLASS_OPTIONS = [
+  'childView',
+  'childViewEventPrefix',
+  'className',
+  'lists',
+];
+
+const CLASS_OPTIONS_ITEM = [
+  'itemClassName',
+  'itemTemplate',
+  'itemTemplateContext',
+];
+
+const Picklist = CollectionView.extend({
+  className: 'picklist__group',
+  tagName: 'li',
+  template: hbs`<ul></ul>`,
+  serializeCollection: noop,
+  childViewContainer: 'ul',
+  childViewEventPrefix: 'item',
+  modelEvents: {
+    'change:query': 'filter',
+  },
+  viewFilter(view) {
+    view.render();
+    const query = this.model.get('query');
+    return !query || !view.searchText || hasAllText(view.searchText, query);
+  },
+  initialize(options) {
+    this.mergeOptions(options, CLASS_OPTIONS_ITEM);
+  },
+  childViewOptions() {
+    const opts = pick(this, ...CLASS_OPTIONS_ITEM);
+    return extend({ state: this.model }, opts);
+  },
+});
+
+const PicklistsCollectionView = CollectionView.extend({
```

Comment:
I trimmed the `PicklistsCollectionView` and `Picklist` components down as much as I knew how here.

Any other ways to trim these? Or re-organize? Open to any suggestions.

Reply:
We could probably add named exports to the component and modify that one.  Or possibly group shared content into an exported mixin.  I'm not entirely sure yet.  This is a complicated one..